library/tasks/GroupMetricsOutputs.wdl

```diff
@@ -1,17 +1,21 @@
+version 1.0
+
 task GroupQCOutputs {
-  Array[File] picard_row_outputs
-  Array[File?] picard_row_optional_outputs
-  Array[File?] picard_table_outputs
-  File hisat2_stats
-  File hisat2_trans_stats
-  File rsem_stats
-  String output_name
-  # Runtime
-  String docker = "quay.io/humancellatlas/secondary-analysis-sctools:v0.3.4"
-  Int mem = 3
-  Int cpu = 1
-  Int disk_space = 20 
-  Int preemptible = 5
+  input {
+    Array[File] picard_row_outputs
+    Array[File] picard_row_optional_outputs
```

Comment:
These no longer need to be optional?

Reply:
I don't know, but I would think its a different ticket to change that. I think it should be in inputs though. I'll fix

Reply:
Oh actually I see what you mean. These can't be optional with WDL 1.0 --this is the issue that Denis helped fix. In practice they never are omitted so this is ok.